tests/ga/test_remoteaccess.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2014 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.4+ and Openssl 1.0+
+#
+
+from tests.tools import *
+from azurelinuxagent.common.protocol.wire import *
+from azurelinuxagent.common.osutil import get_osutil
+
+class TestRemoteAccess(AgentTestCase):
```

Comment:
Please add failure cases.

Reply:
I did not see these, did you add them?

Reply:
Bad data is in the test_remoteaccess_handler test cases since the interesting thing is how the handler reacts to bad data.  Here would just validate an expected exception, nothing more.  I can add another for a completely invalid file though.

---
In reply to: [194028012](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194028012) [](ancestors = 194028012)